elements/pl-file-editor/pl-file-editor.js

```diff
@@ -11,6 +11,8 @@ window.PLFileEditor = function (uuid, options) {
 
   this.inputElement = this.element.find('input');
   this.editorElement = this.element.find('.editor');
+  this.textsizeIncButton = this.element.find(".textsize-inc");
```

Comment:
Since Ace uses "font size", let's use that same terminology instead of "text size".